Code/src/main/java/nl/utwente/group10/ui/components/anchors/ConnectionAnchor.java

```diff
@@ -184,19 +294,24 @@ public final Block getBlock() {
         }
     }
 
-    /** Returns the position of the center of this anchor relative to its pane. */
-    public Point2D getCenterInPane() {
-        Point2D scenePos = localToScene(getCenterX(), getCenterY());
-        return getPane().sceneToLocal(scenePos);
+    /**
+     * @return the position of the center of this anchor relative to its pane.
+     */
+    public Point2D getLocalCenter() {
+        return new Point2D(0, 0);
     }
-
-    /** Returns the pane this anchor resides on. */
-    public final CustomUIPane getPane() {
-        return pane;
+    
+    /**
+     * Reacts to a possible change in active state.
```

Comment:
A bit vague - maybe you could say 'remove all connections if we go from active to inactive state'?